builds/checkin/api-proxy.yaml

```diff
@@ -3,7 +3,14 @@ pr:
   branches:
     include:
       - master
-      - iiot
+      - release/*
+  paths:
+    include:
+      - "edge-modules/api-proxy-module/*"
+
+variables:
```

Comment:
I don't think we need this for PR checks. It is only needed for internal (msazure) pipelines.